simtools/camera_efficiency.py

```diff
@@ -469,12 +508,20 @@ def calc_reflectivity(self):
             [wl_now > 299 and wl_now < 551 for wl_now in self._results["wl"]]
         ]
         c2_sum = np.sum(c2_reduced_wl)
+        cher_spec_weighted_reflectivity = c2_sum / c1_sum / self._results["masts"][0]
 
-        return c2_sum / c1_sum / self._results["masts"][0]
+        return cher_spec_weighted_reflectivity
 
     def calc_nsb_rate(self):
         """
         Calculate the NSB rate.
+
+        Returns
+        -------
+        nsb_rate
+            NSB rate
+        n1_sum
+            Sum of NSB counts
```

Comment:
I agree adding the units makes more sense. I don't know why this function needs to return the counts, since the integral flux is calculated inside it. I havent found any use in the code for the returned counts, I will remove it